### PR TITLE
Added build scripts for windows x86 and x64.

### LINKED ISSUE #9: No 64bit version for Notepad++ v7
Notepad++ v7 [is available](https://notepad-plus-plus.org/download/v7.html) in both 32bit and 64bit editions.

EditorConfig plugin v0.3.1 doesn't work in the latter one.

### PR DESCRIPTION
**Pull request by ViceIce**
Hi,

I've provided build scripts for creating x86 and x64 plugin version.
This schould fix #9

### CODE REVIEW

**Comment by xuhdev** [comment]
Thanks!, awesome! Would mind having relevant scripts into the C core library repository instead of here, and adding some descriptions on how to use these scripts?

**Comment by ViceIce** [comment]
Yes, of cause. I will copy the relevant parts to the core library.

Do we really need the `pcre.lib` file? Shouldn't it be included in the coregion lib?

I do not know much about linking of c or cpp libraries.

**Comment by xuhdev** [comment]
If pcre.lib is statically linked, it is not required to be included in the binary distribution. However, if it is dynamically linked, then it is needed.

**Comment by wangqr** [comment]
```PS
.\init.ps1
.\build.ps1 all -init -install -arch x64
```
The scripts works fine. I tried the above commands and it did everything and put the final dll in `bin\x64\`. I just need to modify the default generator since I only have VS 2017.

**Comment by ViceIce** [comment]
I can make vs version configurable, as I've done in editorconfig/editorconfig-core-c#29.

**Comment by xuhdev** [comment]
Any news :)

**Comment by ViceIce** [comment]
I'll rework and rebase this when  editorconfig/editorconfig-core-c#29 is done. :)

**Comment by xuhdev** [comment]
Into what? BTW, I think `patch.ps1` may be confusing, as some people may have installed the patch program from cygwin.

**Comment by ViceIce** [comment]
`patch.ps1` is a leftover from an older prce lib, wich requires a patch. I 'll remove that. :)

File: appveyor.yml [hunk 1]
```diff
@@ -12,29 +12,29 @@ environment:
...
-  - set PCRE_DEST="%APPVEYOR_BUILD_FOLDER%"\pcre
+  - set PCRE_DEST="%APPVEYOR_BUILD_FOLDER%"\pcre2
   - set CORE_DEST="%APPVEYOR_BUILD_FOLDER%"\editorconfig-core-c
   - set NPP_DEST="%APPVEYOR_BUILD_FOLDER%"
 
   - cmake --version
   - cd "%APPVEYOR_BUILD_FOLDER%"
   # Download PCRE sources
-  - curl -o pcre.zip https://ftp.pcre.org/pub/pcre/pcre-8.41.zip
+  - curl -o pcre.zip https://ftp.pcre.org/pub/pcre/pcre2-10.32.zip
```
Comment (xuhdev) [inline appveyor.yml]: Why not make this version number adjustable?

Comment (ViceIce) [reply appveyor.yml]: Done 😁

**Comment by xuhdev** [comment]
Let me know when you think it's ready :)

**Comment by ViceIce** [comment]
Should AppVeyor use the powershell scripts? Like in editorconfig/editorconfig-core-c#29?

If not, i'm done. 👌

**Comment by xuhdev** [comment]
I think so, so the scripts are always tested. Can you update ApVeyor? 😄

**Comment by ViceIce** [comment]
sure. 😄

**Comment by ViceIce** [comment]
Done. 👌 😁

**Comment by xuhdev** [comment]
Awesome! Thanks!

**Pull request merged**